test/account/deposit.ts

```diff
@@ -19,31 +20,38 @@ describe('Deposit Tests', () => {
     clusterResult1 = await helpers.registerValidators(4, 1, minDepositAmount, helpers.DataGenerator.cluster.new(), [GasGroup.REGISTER_VALIDATOR_NEW_STATE]);
   });
 
-  it('Deposit as owner emits FundsDeposit event', async () => {
+  //Owner deposit
+  it('Owner deposits emits "FundsDeposit"', async () => {
     await helpers.DB.ssvToken.connect(helpers.DB.owners[1]).approve(ssvNetworkContract.address, minDepositAmount);
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[1])['deposit(bytes32,uint256)'](clusterResult1.clusterId, minDepositAmount)).to.emit(ssvNetworkContract, 'FundsDeposit');
-  });
-
-  it('Deposit as non-owner emits FundsDeposit event', async () => {
-    await helpers.DB.ssvToken.connect(helpers.DB.owners[0]).approve(ssvNetworkContract.address, minDepositAmount);
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[0])['deposit(address,bytes32,uint256)'](helpers.DB.owners[1].address, clusterResult1.clusterId, minDepositAmount)).to.emit(ssvNetworkContract, 'FundsDeposit');
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1])['deposit(bytes32,uint256)'](clusterResult1.clusterId, minDepositAmount
+    )).to.emit(ssvNetworkContract, 'FundsDeposit');
   });
 
-  it('Deposit as owner returns an error - ClusterNotExists', async () => {
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[1])['deposit(bytes32,uint256)']('0x392791df626408017a264f53fde61065d5a93a32b60171df9d8a46afdf82992c', minDepositAmount)).to.be.revertedWith('ClusterNotExists');
+  it('Owner deposits gas limits', async () => {
+    await helpers.DB.ssvToken.connect(helpers.DB.owners[1]).approve(ssvNetworkContract.address, minDepositAmount);
+    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[1])['deposit(bytes32,uint256)'](clusterResult1.clusterId, minDepositAmount), [GasGroup.DEPOSIT]);
   });
 
-  it('Deposit as non-owner returns an error - ClusterNotExists', async () => {
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[0])['deposit(address,bytes32,uint256)'](helpers.DB.owners[1].address, '0x392791df626408017a264f53fde61065d5a93a32b60171df9d8a46afdf82992c', minDepositAmount)).to.be.revertedWith('ClusterNotExists');
+  it('Owner deposits reverts "ClusterNotExists"', async () => {
```

Comment:
lets describe whats actually happening here:
1. Owner is not relevant here
2. User trys to deposit to a pod that does not exist
3. User gets a revert of clusterNotExists

We use the word cluster and not pod in the revert cause user doesnt really know what Pod is (this naming will probably be updated later)

Deposit to a not existent cluster reverts "ClusterNotExists"